crates/rnote-ui/src/workspacebrowser/filerow/actions/open_in_default.rs

```diff
@@ -0,0 +1,21 @@
+// Imports
+use crate::workspacebrowser::RnFileRow;
+use crate::RnAppWindow;
+use gettextrs::gettext;
+use gtk4::{gio, gio::prelude::FileExt, glib, glib::clone};
+
+/// Create a new `open-in-default` action.
+pub(crate) fn open_in_default(filerow: &RnFileRow, appwindow: &RnAppWindow) -> gio::SimpleAction {
+    let action_open_in_default = gio::SimpleAction::new("open-in-default", None);
+    action_open_in_default.connect_activate(
+        clone!(@weak filerow, @weak appwindow => move |_action_open_in_default, _| {
+            if let Some(current_file) = filerow.current_file() {
+                if let Err(_) =  open::that(current_file.uri()) {
```

Comment:
Log this error with `log::debug("opening file .. with default app failed, Err: {e:?}" as well here